test/recipe-test.js

```diff
@@ -1,38 +1,38 @@
 import { expect } from 'chai';
 
 import Recipe from '../src/recipe';
-import data from '../data/recipe-data';
```

Comment:
Good catch with the file path naming! It's such a simple thing that can break Everything but they're so hard to find.